index.js

```diff
@@ -173,7 +175,7 @@ module.exports = function(version, _options) {
                 .replace('{lane_instruction}', laneInstruction)
                 .replace('{modifier}', instructions[language][version].constants.modifier[modifier])
                 .replace('{direction}', this.directionFromDegree(language, step.maneuver.bearing_after))
-                .replace('{nth}', nthWaypoint)
+                .replace('{nth}', waypoint)
```

Comment:
nthWaypoint was a more descriptive name. We aren't dealing with the waypoint itself; we're dealing with an ordinal word representing the one-based leg "index".